src/main/Exceptions/IllegalOperationException.java

```diff
@@ -0,0 +1,7 @@
+package main.Exceptions;
```

Comment:
I wonder why your PR wasn't checked by Travis. Please call `mvn clean package` locally.

Reply:
Probably an invalid directory structure, should be working now.